IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Types/IfcGeotechTypicalSectionTypeEnum/DocEnumeration.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcGeotechTypicalSectionTypeEnum" UniqueId="2b1bd183-94e5-4c80-9682-2ef94c48c7b4">
+	<Constants>
+		<DocConstant xsi:nil="true" href="NOTDEFINED_0hF15nkf79uy7wMtnpxz" />
```

Comment:
Do we really have only NOTDEFINED and USERDEFINED?

Reply:
For now.